### DISCUSSION TITLE
How to config url path for blog?

### DISCUSSION THREAD

**Discussion by officialrajdeepsingh** [lumeland/lume#294] [Q&A] [answered]
I recently decided to shift my whole website from Hugo to lume. But I need to achieve the same blog path URL as my old website URL path. 
##### Old website 
* https://officialrajdeepsingh.dev/ -> home page 
* https://officialrajdeepsingh.dev/how-to-install-deno-in-linux/ -> my blog page

How can I achieve the same following pattern with lume? I try this `site.copy("posts",",")` method it is not working.

**Comment by oscarotero** [comment, marked as answer]
Can this help you?
https://github.com/lumeland/lume/discussions/197
